### PR TITLE
Migrate the authenticator documentation from Confluence.

### PR DESCRIPTION
**Pull request by menakaj**
## Purpose
Migrate the EmailOTP authenticator documentation from Confluence. 

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# EmailOTP Authenticator
+ 
+The Email OTP authenticator allows you to authenticate users via email services using Email APIs through WSO2 IS. 
+
```
Comment (Nashaath) [inline docs/README.md:4]: The Email OTP authenticator allows you to authenticate users via email services using email APIs through WSO2 Identity Server.

Comment (menakaj) [reply docs/README.md:4]: Fixed in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/30/commits/0013e9ee1bd5e1a1d1d4f5970d968e1ed2c0be91

File: docs/html-support.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# Enable HTML Templates in EmailOTP
+
+Use this configurations if you need to use HTML Templates in the Emails.
+**This feature is supported from Identity Server v5.6 onwards**.
+
```
Comment (Nashaath) [inline docs/html-support.md:4]: Shall we say:

Follow the steps below if you want to use HTML templates in emails when you use the Email OTP authenticator.
**NOTE**: The use of HTML templates is only supported from WSO2 Identity Server version 5.6.0 onwards.

File: docs/html-support.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+1. Configure Email settings in \<IS_HOME>/repository/conf/output-event-adapters.xml file.
```
Comment (Nashaath) [inline docs/html-support.md]: Can we say:
Open the <IS_HOME>/repository/conf/output-event-adapters.xml file and configure the appropriate email settings as follows:

File: docs/html-support.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    <property key="mail.smtp.starttls.enable">true</property>
+    <property key="mail.smtp.auth">true</property>
+    <!-- Thread Pool Related Properties -->
+    <property key="minThread">8</property>
+    <property key="maxThread">100</property>
+    <property key="keepAliveTimeInMillis">20000</property>
+    <property key="jobQueueSize">10000</property>
+</adapterConfig>
+```
+2. Open \<IS_HOME>/repository/conf/identity/application-authentication.xml and add the following parameter into
```
Comment (Nashaath) [inline docs/html-support.md:27]: Shall we say:
Open the <IS_HOME>/repository/conf/identity/application-authentication.xml and add the following parameter under <AuthenticatorConfig name="EmailOTP" enabled="true">:

Comment (menakaj) [reply docs/html-support.md:27]: Fixed in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/30/commits/f578717bad3139812d57d434f1d4275141d260be

File: docs/html-support.md (hunk 3 above)
Comment (menakaj) [reply docs/html-support.md]: Fixed in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/30/commits/f578717bad3139812d57d434f1d4275141d260be

**Review by Kanapriya — APPROVED** [review]

**Review by Nashaath — APPROVED** [review]

**Pull request merged**